pkg/translate/output.go

```diff
@@ -79,16 +80,24 @@ func printDisconnectedBlocks(wr io.Writer, root *common.Vertex, domainVisited ma
 	return nil
 }
 
-func verifyBlockSize(domainVisited map[string]int) int {
-	blockSize := -1
-	for _, bSize := range domainVisited {
-		if blockSize == -1 {
-			blockSize = bSize
-		} else if blockSize != bSize {
-			fmt.Printf("Alert! blockSize different between NVL domains\n")
+func getBlockSize(domainVisited map[string]int, adminBlockSize string) string {
+	minDomainSize := -1
+	for _, dSize := range domainVisited {
+		if minDomainSize == -1 || minDomainSize > dSize {
+			minDomainSize = dSize
 		}
 	}
-	return blockSize
+	if adminBlockSize != "" {
+		blockSizes := strings.Split(adminBlockSize, ",")
```

Comment:
check that len(blockSezes) == 2 ?

Reply:
If ',' is not present, then Split will return entire adminBlockSize which means only 1 blockSize is entered by the admin. In either case, we need access to only blockSizes[0]. So checking for length 2 is not necessary IMO

Reply:
ok, fair